### ISSUE TITLE
Additional Actions Modal

### ISSUE THREAD

**Issue by sarahscott** [radicle-dev/radicle-upstream#193]
Ellipses button + corresponding modal next to 'Add project':
![Screen Shot 2020-02-26 at 1 12 35 PM](https://user-images.githubusercontent.com/2712962/75388403-bea5d100-5899-11ea-8c51-cc85d4448baa.png)

![Screen Shot 2020-02-26 at 1 10 26 PM](https://user-images.githubusercontent.com/2712962/75388400-bc437700-5899-11ea-8d23-640e3464a1b1.png)

**Comment by rudolfs** [comment]
I think this issue should be about a generic component, which we can reuse in multiple places. I'd rephrase the issue title to something like: "MoreActions dropdown component".
